_docs/error-handling.md

```diff
@@ -46,7 +46,7 @@ We categorize most errors into the following categories:
 Occasionally a transaction has neither errored, nor has paid out. We call these transactions pending. There are two main cases for pending transactions:
 
 * For any kind of cash pickup transactions, pending means that the recipient has received a pickup notice, but didn't pick up the funds yet. These transactions can be cancelled, in which case the pickup notice will be invalidated and the recipient will not be able to use it to pick up the funds anymore.
-* For other transactions pending means that the recipient's bank or mobile provider is still processing the transaction and will note whether it could deposit the funds or not. You cannot cancel these transactions, which is shown on the recipient as well, where the `may_cancel` field will be `false`.
+* For other transactions pending means that the recipient's bank or mobile provider is still processing the transaction and will note whether it could deposit the funds or not. These transactions, where `may_cancel` field will be `false` can still be cancelled, however they will get cancelled the next available time window after you have called the `DELETE /v1/recipients/[TRANSFERZERO_RECIPIENT_ID]` endpoint.
```

Comment:
```suggestion
* For other transactions pending means that the recipient's bank or mobile provider is still processing the transaction and will note at a later date whether it could deposit the funds or not. These transactions will have a `may_cancel` field set to `false`. If you try to cancel them using the `DELETE /v1/recipients/[TRANSFERZERO_RECIPIENT_ID]` endpoint they get cancelled right after we get confirmation from our bank or mobile provider that the funds could not be debited. However if the confirmation we receive says that the funds are sent out these transactions will not be cancelled, they will be marked as successful.
```